api/go.mod

```diff
@@ -67,9 +67,9 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.29.0 // indirect
-	k8s.io/client-go v0.29.0 // indirect
-	k8s.io/component-base v0.29.0 // indirect
+	k8s.io/apiextensions-apiserver v0.28.7 // indirect
+	k8s.io/client-go v0.28.7 // indirect
+	k8s.io/component-base v0.28.7 // indirect
 	k8s.io/klog/v2 v2.110.1 // indirect
 	k8s.io/kube-openapi v0.0.0-20240103051144-eec4567ac022 // indirect
```

Comment:
we should have downgraded this too, as this is again points to a module needing golang 1.21

Reply:
being fixed in https://github.com/openstack-k8s-operators/dataplane-operator/pull/724

Reply:
and a version pin is added for cert-manager in our renovate config  https://github.com/openstack-k8s-operators/renovate-config/pull/15